homework/homework_20/homework_20_1.py

```diff
@@ -0,0 +1,97 @@
+"""Homework # 20.1"""
+import unittest
+import logging
+from homework.homework_11 import homework_11_1
+
+formatter = logging.Formatter('[%(asctime)s] %(levelname)s - %(message)s')
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+
+logger = logging.getLogger(__name__)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
```

Comment:
You can use 1 logger across a single application, e.g.: https://stackoverflow.com/questions/75489656/how-to-use-a-single-python-logging-object-throughout-the-python-application

Reply:
but you already have it in the `logger_module.py`, so it here as well